klippy/extras/raspberry_pi_temperature.py

```diff
@@ -0,0 +1,62 @@
+# Support for Raspberry Pi temperature sensor
+#
+# Copyright (C) 2020  Al Crate <al3ph@users.noreply.github.com>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+import logging
+
+RASPBERRYPI_REPORT_TIME = 1.0
+RASPBERRYPI_MIN_REPORT_TIME = 0.5
+
+class RaspberryPiTemperature:
+    def __init__(self, config):
+        self.printer = config.get_printer()
+        self.name = config.get_name().split()[-1]
+        self.reactor = self.printer.get_reactor()
+        self.report_time = config.getint(
+            'raspberry_pi_temperature_report_time',
+            RASPBERRYPI_REPORT_TIME,
+            minval=RASPBERRYPI_MIN_REPORT_TIME
+        )
```

Comment:
Oh - as an additional comment - is it necessary to make this configurable?  I think we should be able to figure out a reasonable default sampling time.

-Kevin

Reply:
Yup once a second should be fine.